src/components/widgets/status/StatusTab.vue

```diff
@@ -71,7 +71,7 @@
               </status-label>
 
               <status-label :label="$t('app.general.label.filament')">
-                <span v-if="filament_used > 0 && printerPrinting">{{ $filters.getReadableLengthString(filament_used) }}</span>
+                <span v-if="filament_used > 0 && filament_total > 0 && printerPrinting">{{ $filters.getReadableLengthString(filament_used) }} / {{ $filters.getReadableLengthString(filament_total) }} <br /> remaining: {{filament_weight_needed.toFixed(2)}} g  </span>
```

Comment:
so if `filament_total` is not `> 0`, this won't display at all now?

not sure we would want to hide it if so.  could we remove the condition `&& filament_total > 0`?